addons/account/views/account_payment_term_views.xml

```diff
@@ -108,10 +108,12 @@
                     <field name="name"/>
                     <field name="note"/>
                     <templates>
-                        <t t-name="kanban-box">
-                            <div t-attf-class="oe_kanban_global_click">
-                                <div><strong class="o_kanban_record_title"><t t-out="record.name.value"/></strong></div>
-                                <div t-if="!widget.isHtmlEmpty(record.note.raw_value)"><t t-out="record.note.value"/></div>
+                        <t t-name="kanban-card">
+                            <div>
+                                <strong class="h5"><field name="name"/></strong>
+                            </div>
+                            <div>
+                                <field name="note"/>
```

Comment:
```suggestion
                        <t t-name="kanban-card">
                            <strong class="h5"><field name="name"/></strong>
                            <field name="note"/>
                        </t>
```

to try

Reply:
I tried this, but I'm getting more space than expected. I wrapped it in one div instead of two, and it works fine that way.

Reply:
```xml
<field class="fw-bold fs-5" name="name"/>
<field name="note"/>
```
`fw-bold` is a bit less bolder than before, but `fw-bolder` is too much, so I would go with `fw-bold` (I think it's fine to have such slight differences that no one will notice). Also in this case, it's clearly not an `h5`, as `h5` comes with a bottom margin which we don't want...

At the end, we'll (hopefully) have consistency between all kanban if we do the same choices everywere (i.e. we want a gap between "header" and the rest => `h5`, we don't want a gap but we want the "header" to be bold => `fw-bold fs-5`).